superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -343,3 +343,58 @@ def _timer(self):
         if self._gpu_available:
             torch.cuda.synchronize()
         return time.time()
+
+    def _benchmark(self):
+        """Wrap super._benchmark with profiler context if enabled by environment variable.
+
+        Set SB_PYTORCH_PROFILER='1' to enable profiling.
+        """
+        # Check if this is a Nvidia GPU
+        if not (torch.cuda.is_available() and torch.version.cuda is not None):
+            return super()._benchmark()
+
+        # Check if profiling is enabled via environment variable
+        enable_profiler = os.environ.get('SB_ENABLE_PYTORCH_PROFILER', '0').lower() == '1'
```

Comment:
Why call `lower()` here? and doesn't call for `enable_nsys = os.environ.get('SB_ENABLE_NSYS', '') == '1'`?